src/pages/Departments/index.ts

```diff
@@ -1,3 +1,3 @@
 import { lazy } from "react";
 
-export const DepartmentsPage = lazy(() => import("./departments.page"));
+export const DepartmentsPage = lazy(() => import("./Departments.page"));
```

Comment:
Departments should be lowercase as any other pages names

Reply:
tweaked